hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestUtils.java

```diff
@@ -313,7 +313,7 @@ public static Option<HoodieCommitMetadata> getCommitMetadataForLatestInstant(Hoo
    * @return a new {@link HoodieTableMetaClient} instance.
    */
   public static HoodieTableMetaClient createMetaClient(JavaSparkContext jsc, String basePath) {
-    return HoodieTestUtils.createMetaClient(jsc.hadoopConfiguration(), basePath);
+    return HoodieTestUtils.createMetaClient(HoodieStorageUtils.getStorageConf(jsc.hadoopConfiguration()), basePath);
```

Comment:
Still keep a test util `HoodieTestUtils.createMetaClient` that takes Hadoop configuration?  Or use `HoodieClientTestUtils#createMetaClient(JavaSparkContext jsc, String basePath)`?

Reply:
Meta client is is hudi-common so it will take in storage conf. HoodieTestUtils is also in hudi-common. I don't see what the problem is with my change

Reply:
I was thinking about reusing utils.  Did not notice this method is within `HoodieClientTestUtils`.  I see other places are fixed.